organization/_posts/2021/2021-03-18-coordination.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "HSF Weekly Meeting #204, 18 March 2021"
+layout: plain_toc
+---
+
+Present: Graeme Stewart, Benedikt Hegner, Serhan Mete, Eduardo Rodrigues, Nicola Skidmore, Mason Proffitt, Chris Jones, Andi Salzburger, Andrei Gheata, Serhan Mete, Paul Laycock, Stefan Roiser, Allie Hall, Dorothea vom Bruch, Pere Mato, Efe Yazgan, Kevin Pedro, Ben Morgan, Liz Sexton-Kennedy, Michel Jouvin, Teng Jian Khoo, Meirin Oan Evans, Daniel Elvira, Marc Paterno, David Lange, Andrew Norman, Philippe Canal
+
+Apologies: Krzysztof Genser
+  
+## News, general matters, announcements
+
+### LHCC
+
+The software WLCG report for 2021H1 was prepared - see Indico agenda.
+
+Thanks to all who made comments and suggestions.
+
+### HL-LHC Review Phase 2
+
+We have had some meetings with key people for each section of the document. Now making progress in contacting other projects and arranging mini-workshops, as needed.
+
+### Google Summer of Code, Season of Docs
+
+The GSoC mentors are quite busy at the moment with candidate evaluation and there are few exchanges on the mentors list.
+
+For GSoD so far we have **no applications**. *Do no projects need documentation help?* If no proposals we won't apply. Deadline is 25 March.
+
+- Will send another reminder
+- Can we consider also facility areas that need documentation?
+    - Yes
+
+### November Workshop Survey and Outcomes
+
+See slides attached to the agenda, which give highlights of the survey from the workshop.
+
+- Feedback: no need for a single virtual workshop to be packed with disparate topics, so indeed perhaps better to split into smaller events.
+
+### Proposal for DUNE Framework Requirements Review
+
+Paul: DUNE is trying to workout how to deal with its data, including massive "events" from supernovae (100GB). DUNE task force has defined the requirements (43 of them) in a document, which is aimed at DUNE people. If the HSF is giving feedback on this, how can it be expanded/annotated to non-DUNE people? Once that's done can we get together a group of experts who could review and bring input from the expertise of the wider community?
```

Comment:
A supernova "event" is over 100TBs, believe it or not, it's 100 seconds of continuous readout.